### ISSUE TITLE
"Add Library" select element should use optgroup

### ISSUE THREAD

**Issue by karlgroves** [jsbin/jsbin#1920]
`//*[@id="library"]`

Currently JSBin uses a visual separator to distinguish between families of JS libraries.

For instance: `<option value="" data-group="jQuery" class="heading">-------------</option>`

Instead this should be:

```
<optgroup label="jQuery">
<!--Individual options go here-->
</optgroup>
```

This will allow for more accessible experience and allow users to understand the groupings more easily.

**Comment by Martin-Pitt** [comment]
Would something like this be better?  (Excuse the crappy mockup)

http://jsbin.com/givuli/edit?output


(Press 'Add library', add jQuery;  Reset bin via "Run with JS" button)

**Comment by karlgroves** [comment]
@Martin-Pitt Getting there. The keyboard accessibility seems to work well, but as you arrow through the choices the options aren't announced to the user.

jQuery UI autocomplete uses an ARIA live region to announce options as you arrow through.  If you'd like I can write up a Gherkin-style feature spec for this.
